conf/mini_provisioner/v2/ha.cleanup.tmpl.3-node

```diff
@@ -10,13 +10,33 @@ cortx:
       secret: TMPL_SECRET
       user: TMPL_USER
 server_node:
-  TMPL_MACHINE_ID:
-    cluster_id: TMPL_CLUSTER_ID
+  TMPL_MACHINE_ID-1:
```

Comment:
Lets change to TMPL_MACHINE_ID_1
And for all below.

Reply:
Done